src/views/app/collections.js

```diff
@@ -103,22 +105,25 @@ class CollectionsPage extends Component {
           {
             collections && collections.length > 0 ?
               (collections.map((item, index) => {
-                return (
-                  <Colxx xxs="12" xs="6" md="4" lg="3" key={index}>
-                    <Link to={`/app/collection/${item.id}/general`}>
-                      <Card className="mb-2">
-                        <CardBody>
-                          <div>
-                            <h5>{item.name}</h5>
-                          </div>
-                          <div>{moment(item.createdAt).format("MM/DD/YYYY hh:mm")}</div>
-                          <div>{item.tokenCount} tokens</div>
-                          {user && user === item.userId ? (<Badge>My collection</Badge>) : ""}
-                        </CardBody>
-                      </Card>
-                    </Link>
-                  </Colxx>)
-              })) :
+                if (item.cards.length > 0){
+                  return (
+                    <Colxx xxs="12" xs="6" md="4" lg="3" key={index} >
+                      <Link to={`/app/collection/${item.id}/general`}>
+                        <Card className="mb-2">
+                          <CardBody style={{opacity: "0.4",backgroundImage:`url('https://ipfs.io/ipfs/${item.cards[0].dataLink}')`, backgroundSize:"cover",backgroundPosition: "center top",}}>
```

Comment:
Avoid using inline styles, opacity should not be used on content blocks

Reply:
here my doubt is that we don't use inline styles or just to remove opacity in inline styles?

Reply:
Both